src/bitcoin-cli.cpp

```diff
@@ -704,7 +704,8 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "           \"b\" - BLOOM: peer can handle bloom-filtered connections (see BIP 111)\n"
         "           \"w\" - WITNESS: peer can be asked for blocks and transactions with witness data (SegWit)\n"
         "           \"c\" - COMPACT_FILTERS: peer can handle basic block filter requests (see BIPs 157 and 158)\n"
-        "           \"l\" - NETWORK_LIMITED: peer limited to serving only the last 288 blocks (~2 days)\n"
+        "           \"l\" - NETWORK_LIMITED: peer can serve at least the last 288 blocks (~2 days); signals a pruned or otherwise\n"
+        "                                  limited peer (i.e., assumeutxo download in progress) if NETWORK is unset (see BIP 159)\n"
```

Comment:
We should probably try to keep the output lines of `bitcoin-cli -netinfo help` below 80 characters. It doesn't wrap in a nice way.

Reply:
Don't disagree but that would require reformatting most of the help.

Reply:
For here kept it shorter than the longest line ("If that argument is passed...")

Reply:
Shortened the line length in the last push.

Reply:
Just the line you're touching is fine, yes. Thanks.